setup.py

```diff
@@ -24,7 +24,7 @@
         #   50.0 is broken: https://github.com/pypa/setupatools/issues/2353
         "setuptools>=41.0.0,!=50.0",
         "cached-property>=1.5.2",
-        "click>=8.0.3",  # used in scl
+        "click==8.0.4",  # used in scl
```

Comment:
I think we could use the most recent `8.0` version.

```suggestion
        "click~=8.0",  # used in scl
```